### ISSUE TITLE
Expose both security events and journal records on project history

### ISSUE THREAD

**Issue by woodruffw** [pypa/warehouse#6425]
Once #6339 is merged, the "History" tab for project management will contain the event security log instead of the project journal. We should look at re-exposing the project journal, possibly as a second table on that same tab.

cc @nlhkabu

**Comment by nlhkabu** [comment]
I would appreciate your feedback on this @ewdurbin and @dstufft, as I understand that this was briefly discussed on Slack.

@brainwane - would you consider this part of the security milestone, or out of scope?

**Comment by brainwane** [comment]
@nlhkabu Since the project journal at `warehouse/templates/manage/history.html` currently displays new releases and removed releases, and since #6339 [does not include that info](https://github.com/pypa/warehouse/pull/6339#issuecomment-520911074), I think we have 2 choices:

a: don't merge #6339 yet -- wait until we have a new PR that adds events for upload or removal of a specific release/file to the event security log, then merge #6339 and then immediately merge the new PR
b. re-expose the project journal (possibly via a rename from "history" to something else)

@woodruffw @nlhkabu which of these do you think will take less time/redundancy?

**Comment by woodruffw** [comment]
The second option (re-exposing the journal) would probably take less time; I could do it as part of #6339. OTOH it would take about the same amount of time to just add those events to the audit log, so I could do *that* instead (and that would fit more cleanly into the changeset).

**Comment by nlhkabu** [comment]
My preference here would be to add those events to the audit log to avoid a temporary situation where we have duplicate "history" tables.

**Comment by ewdurbin** [comment]
I think it's reasonable to maintain both tables. Journals could be noted as effectively an implementation detail that will be replaced by the Security History.

**Comment by nlhkabu** [comment]
> Journals could be noted as effectively an implementation detail that will be replaced by the Security History

You mean that we should add some help text or a badge to explain that we plan on deprecating this?  That works for me :)

**Comment by brainwane** [comment]
OK, that's fine with me.

**Comment by woodruffw** [comment]
Added the remaining events in #6339 and re-added the old journal table as a separate tab, so closing this.
